### ISSUE TITLE
Too many series returned when using rolled-up data and specifying single tag key

### ISSUE THREAD

**Issue by overstre** [OpenTSDB/opentsdb#1083]
When using rolled-up data in 2.4RC2, specifying a single tag value directly returns multiple series when 1 series was expected.  Putting tag value inside of regexp returns 1 series for rolled-up data as expected.  Using data from default interval also works as expected.

Test data was sent using the api/rollup endpoint for multiple timestamps.  
```     
   test_data = [{"metric": "sys.cpu.nice",
                      "timestamp": ts,
                      "value": 18,
                      "tags": {"host": "web01",
                               "dc": "lga"},
                      "interval": "1h",
                      "aggregator": "SUM"},
                     {"metric": "sys.cpu.nice",
                      "timestamp": ts,
                      "value": 9,
                      "tags": {"host": "web02",
                               "dc": "lga"},
                      "interval": "1h",
                      "aggregator": "SUM"}]
```

Query was
```
/api/query?start=2d-ago&m=sum:1h-sum-none:sys.cpu.nice{host=web01}
```
response was
```
[{"metric":"sys.cpu.nice","tags":{"host":"web01","_aggregate":"RAW","dc":"lga"},"aggregateTags":[],"dps":{"1507762800":18.0,<truncated>}},{"metric":"sys.cpu.nice","tags":{"host":"web02","_aggregate":"RAW","dc":"lga"},"aggregateTags":[],"dps":{"1507762800":9.0,<truncated}}]
```

**Comment by manolama** [comment]
I'll check this one out before cutting it.

**Comment by manolama** [comment]
Found it and fixed in d342301898ccc3edaee18d711a6d9684007d16b8
